kratos/tests/cpp_tests/geometries/test_removing_warnings_nurbs_utilities.cpp

```diff
@@ -0,0 +1,59 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Andreas Apostolatos
+//
+
+// System includes
+#include <limits>
+
+// External includes
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/pointer_vector.h"
+#include "geometries/nurbs_shape_function_utilities/nurbs_utilities.h"
+
+#include "tests/cpp_tests/geometries/test_geometry.h"
+
+namespace Kratos {
+namespace Testing {
+
+        ///// Tests
+        KRATOS_TEST_CASE_IN_SUITE(removingWarningsNurbsUtilities, KratosCoreNurbsGeometriesFastSuite) {
+            
+            // Dummy polynomial order
+            const int polynomial_degree = 1;
+
+            // Dummy number of knots
+            const int number_of_knots = 2;
+
+            // Dummy number of control points
+            const int number_of_control_points = 2;
+            
+            // Dummy knot vector
+            Vector knot_vector = ZeroVector(number_of_knots);
+            knot_vector[0] = 0.0;
+            knot_vector[1] = 1.0;
+
+            // Dummy curve parameter
+            double parameter_t = 0.5;
+
+            // Call the functions from the geometries/nurbs_shape_function_utilities/nurbs_utilities.h to remove the warning
+            NurbsUtilities::GetNumberOfSpans(polynomial_degree, number_of_knots);
```

Comment:
wouldn't it make sense to also test against a value?

Reply:
@philbucher suggested to just call the functions in order to skip the warning, but now that you mention that it does really make sense :) thanks!

Reply:
Could be just a simple wrapper around it. But, then at least it is really a test. ;) Thanks!